x-pack/plugins/security_solution/public/common/components/visualization_actions/utils.ts

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Filter } from '@kbn/es-query';
+
+export const getHostDetailsPageFilter = (hostName?: string): Filter[] =>
+  hostName
+    ? [
+        {
+          meta: {
+            index: 'e5bb994d-e8fb-4ddb-a36e-730ad8cc0712',
```

Comment:
what is this `index`? seems strange to have an id here

Reply:
agree, I should have removed this. Removed in https://github.com/elastic/kibana/pull/126507/commits/a289c86e03de7dd05b2566ab5abbcc1b2b320f91